### ISSUE TITLE
How to get the current leverage of Binance's ETFs?

### ISSUE THREAD

**Issue by CF524** [ccxt/ccxt#8177]
I am attempting to get the current leverage for ETFs listed on Binance by querying the API via ccxt. After reviewing the docs, it is still not clear to me if this is possible or how one should go about doing it.

For the sake of example, I have been looking at the 'BTCUP/USDT' symbol and attempting to extract its leverage. At the time of writing it is 2.152x according to its entry on the Binance website.

Is there a call that can be placed to obtain this elegantly? Otherwise, I assume that the best way to obtain the figure is by comparing the returns of this symbol and its underlying to calculate leverage.

Any help would be greatly appreciated. Thanks in advance.

**Comment by kroitor** [comment]
Yes, you can do that using the implicit endpoints as described in the Manual:

- https://github.com/ccxt/ccxt/wiki/Manual#implicit-api-methods

What is your programming language?

**Comment by CF524** [comment]
I am working in Python and using ccxt v1.39.90. Thanks for pointing me towards the correct part of the Manual. This has allowed me to find the solution.

I was able to access these figures for all of Binance's leveraged tokens by calling `exchange.sapiGetBlvtTokenInfo()` where `exchange` is an instance of ccxt's Binance exchange class.

For the example I gave in the question, the leverage can be found by
```
btcup_info = exchange.sapiGetBlvtTokenInfo({'tokenName': 'BTCUP'})[0]
btcup_leverage = btcup_info['realLeverage']
```
Thanks for your help!
